Mixpanel/DecideRequest.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  DecideRequest.swift
+//  Mixpanel
+//
+//  Created by Yarden Eitan on 8/5/16.
+//  Copyright © 2016 Mixpanel. All rights reserved.
+//
+
+import Foundation
+
+class DecideRequest: Network {
+
+    typealias DecideResult = [String: AnyObject]
+    let decidePath = "/decide"
+    var networkRequestsAllowedAfterTime = 0.0
+    var networkConsecutiveFailures = 0
+
+    struct DecideQueryItems {
+        let version: URLQueryItem
```

Comment:
don't we also send `ios_app_version`, `ios_lib_version` and some more params on every decide request?